### ISSUE TITLE
San Andreas (classic) crashing after a few minutes of booting up.

### ISSUE THREAD

**Issue by Sbo-RSA** [doitsujin/dxvk#4233]
Please describe your issue as accurately as possible.

If you use Windows, please check the following page: https://github.com/doitsujin/dxvk/wiki/Windows

### Software information
Name of the game, settings used etc.

### System information
- GPU: AMD Radeon R3
- Driver: 2.0.179
- Wine version: 1.2.170
- DXVK version: 1.9.4

### Apitrace file(s)
- Put a link here

For instructions on how to use apitrace, see: https://github.com/doitsujin/dxvk/wiki/Using-Apitrace

### Log files
Please attach Proton or Wine logs as a text file:
- When using Proton, set the Steam launch options for your game to `PROTON_LOG=1 %command%` and attach the corresponding `steam-xxxxx.log` file in your home directory.
- When using regular Wine, use `wine game.exe > game.log 2>&1` and attach the resulting `game.log` file.
[gta_sa_d3d9.log](https://github.com/user-attachments/files/16852252/gta_sa_d3d9.log)

**Comment by Blisto91** [comment]
Ho there.
We don't support old versions of dxvk. The best you can do is try 1.10.3 which is the latest release that will work with your driver.

**Comment by Sbo-RSA** [comment]
> Hi there. We don't support old versions of dxvk. The best you can do is try 1.10.3 which is the latest release that will work with your driver.

I've tried various versions including 1.10.3 and they all don't work. When I use some them the game doesn't launch at all. 
[gta_sa_d3d9.log](https://github.com/user-attachments/files/16858686/gta_sa_d3d9.log)

**Comment by Sbo-RSA** [comment]
I've also noticed some lines that usually don't appear when using DXVK on other gmes. I'm not sure if this is useful or not but I thought I should mention it.
![image](https://github.com/user-attachments/assets/91da1cb7-b2d6-4e9d-82ab-dbfeeb498e29)

![image](https://github.com/user-attachments/assets/323057e4-7464-46e7-a77e-3cbc52e6b953)

**Comment by dungeon007** [comment]
These likely arent DXVK crashes, but GPU hangs... of a year 2021.Q1 era drivers.
Just played a bit of TopSpin and AMDVLK 2021.Q1.6 driver hanged after couple minutes too... so you cant help it :dagger: 
![2021 Q1](https://github.com/user-attachments/assets/e5e8c658-e73e-4d00-98a5-ca142fcecd15)

**Comment by Sbo-RSA** [comment]
Do those work on Windows? From what I can find RADV doesn't work on Windows.

**Comment by RouatbiH** [comment]
Also not working in windows with DXVK 2.4
err:   D3D9SwapChainEx: Failed to recreate swap chain: VK_ERROR_INITIALIZATION_FAILED
[gta_sa_d3d9.log](https://github.com/user-attachments/files/16870439/gta_sa_d3d9.log)

**Comment by Sbo-RSA** [comment]
> Also not working on Windows with DXVK 2.4 err: D3D9SwapChainEx: Failed to recreate swap chain: VK_ERROR_INITIALIZATION_FAILED [gta_sa_d3d9.log](https://github.com/user-attachments/files/16870439/gta_sa_d3d9.log)

I saw from one post claiming that the issue can be solved with changing a single line. I'm not sure what line that is since I don't know the first thing about coding, sadly.

**Comment by Blisto91** [comment]
@Sbo-RSA I don't think there is much for us to do here since your setup doesn't support Vulkan 1.3. dxvk have already received a whole bunch of 32bit memory and vram (useful on say a Laptop that might system memory for vram) improvements. I assume it is a out of memory issue.

 @RouatbiH Please open a new issue as that doesn't seem related.

**Comment by Sbo-RSA** [comment]
@Blisto91 I just realised that I've made a mistake. I'm actually using Windows 10, not Vulkan. I've been trying anything supported by Vulkan 1.2 and none of them seem to work. it always ends with the game locking up for a few minutes and sometimes indefinitely til I restart it. If it's a memory issue, is it possible to adjust that through dxvk.config? Because I've tryied halving the amount but I still experienced crashes. I onl have 8 GB and usually only 5 GB is usable. Oh and I should mention that I've modified the exe file to use 4 GB RAM. Could that also be the cause?
